src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn submission-kill [workspace-id submission-id]
```

Comment:
Is it redundant to add "-path" as a suffix because it's already in a file named paths? (and by this logic all the -path suffixes above are also redundant)